src/strings/PowerFxResources.en-US.resx

```diff
@@ -6517,4 +6517,55 @@
     <value>value</value>
     <comment>function_parameter - Second argument to the Set function - any Power Apps value.</comment>
   </data>
+  <data name="AboutDec2Hex" xml:space="preserve">
+    <value>Converts a decimal number to hexadecimal</value>
+    <comment>Description of 'Dec2Hex' function.</comment>
+  </data>
+  <data name="Dec2HexArg1" xml:space="preserve">
+    <value>number</value>
+  </data>
+  <data name="AboutDec2Hex_number" xml:space="preserve">
+    <value>A numeric decimal value to convert to hexadecimal string.</value>
+    <comment>Description of 'Dec2HexArg1' function parameter.</comment>
+  </data>
+  <data name="Dec2HexArg2" xml:space="preserve">
+    <value>number</value>
```

Comment:
>number [](http://example.com/codeflow?start=11&length=6)

Should be `places` instead of `number` ('number' is the first argument to this function)